NOTICE.txt

```diff
@@ -65,12 +65,7 @@ by Adobe Systems Incorporated:
 Modified redistributions
 ========================
 
-This product contains a modified part of Brave, distributed by Zipkin.io:
-
-  * License: licenses/LICENSE.brave.al20.txt (Apache License v2.0)
-  * Homepage: https://github.com/openzipkin/brave/
```

Comment:
I believe this referred to this code that's still there - don't think I removed any forked code in the brave 4.5 update

https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/logging/CountingSampler.java